src/lib.rs

```diff
@@ -25,8 +27,16 @@ pub enum Error {
     Unknown(NTSTATUS),
 }
 
+impl fmt::Display for Error {
+    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
+        fmt::Debug::fmt(self, f)
```

Comment:
I think that [`write!`](https://doc.rust-lang.org/std/macro.write.html) is recommended here.

Reply:
I wanted to be crystal clear that we simply delegate to `Debug` trait here.

Reply:
Changed the implementation to use `write!`.